README.md

```diff
@@ -202,10 +203,9 @@ TODO: Add section about `documentLoader`
 To create a verifiable presentation out of one or more verifiable credentials:
 
 ```js
-const verifiableCredential = [vc1, vc2];
-
-const vp = await vc.createPresentation({verifiableCredential, suite});
-console.log(JSON.stringify(vp, null, 2));
+options = { 'verifiableCredential': credential, suite, 'type': 'AlumniCredential' };
```

Comment:
Key quotes not needed.
```suggestion
options = {verifiableCredential: credential, suite, type: 'AlumniCredential'};
```

Reply:
@davidlehn 69255c3 should do it; let me know how this looks...